crates/tauri/src/window/scripts/drag.js

```diff
@@ -11,12 +11,18 @@
   // moves after the double click, it should be cancelled (see https://github.com/tauri-apps/tauri/issues/8306)
   //-----------------------//
   const TAURI_DRAG_REGION_ATTR = 'data-tauri-drag-region'
-  let x = 0,
-    y = 0
+
+  // initial mousedown position for macOS
+  let initialX = 0
+  let initialY = 0
+
   document.addEventListener('mousedown', (e) => {
+    const attr = e.target.getAttribute(TAURI_DRAG_REGION_ATTR)
     if (
       // element has the magic data attribute
-      e.target.hasAttribute(TAURI_DRAG_REGION_ATTR)
+      attr
+      //  and not false
+      && attr !== 'false'
```

Comment:
This should probably be 

```suggestion
      e.target.hasAttribute(TAURI_DRAG_REGION_ATTR)
      //  and not false
      && e.target.getAttribute(TAURI_DRAG_REGION_ATTR) !== 'false'
```

`e.target.getAttribute(TAURI_DRAG_REGION_ATTR)` will return an empty string when used like this `<div data-tauri-drag-region></div>

Reply:
ah right, we can just check for null then, since that's what we care about
```suggestion
      attr !== null
      //  and not false
      && attr !== 'false'
```